### PR TITLE
Schemas: Improve list inputs for tuples

### PR DESCRIPTION
**Pull request by pleek91**
# Description
Fixes bug when deleting items from a list. Also prevents adding more values to a list when it has `maxItems`. 

image

### CODE REVIEW

File: src/schemas/components/SchemaFormPropertyArray.vue [hunk 1]
```diff
@@ -21,6 +21,11 @@
         />
       </template>
     </p-draggable-list>
+    <template v-if="!allowCreate && property.maxItems">
```
Comment (collincchoy) [inline src/schemas/components/SchemaFormPropertyArray.vue:24]: nit: probably nbd but if for some reason `property.maxItems === 0`, you wouldn't be able to add anything and you wouldn't ever see the message

File: src/schemas/components/SchemaFormPropertyArray.vue [hunk 2]
```diff
@@ -100,4 +112,10 @@
   italic;
   margin-left: 30px;
 }
+
+.schema-form-property-array__max { @apply
+  text-subdued
+  text-sm;
+  margin-left: 30px;
```
Comment (collincchoy) [inline src/schemas/components/SchemaFormPropertyArray.vue:119]: nit: css variable for this magic #?

**Review by collincchoy — APPROVED** [review]
LGTM

> Fixes bug when deleting items from a list.
What was the bug/fix for deletion? jc

File: src/schemas/components/SchemaFormPropertyArray.vue (hunk 2 above)
Comment (pleek91) [reply src/schemas/components/SchemaFormPropertyArray.vue]: Good suggestion!

**Comment by pleek91** [comment]
> LGTM
> 
> > Fixes bug when deleting items from a list.
> > What was the bug/fix for deletion? jc

Sorry, terrible PR description on my part. When you deleted an item the updated list was doing weird stuff because of the `keep-alive`. Things would be out of place and sometimes the list length wouldn't change.

**Pull request merged**